libs/core/src/lib/pagination/pagination.component.html

```diff
@@ -14,23 +14,24 @@
                 (keypress)="onKeypressHandler(currentPage - 1, $event)"
                 (click)="previousPage()"
             >
-            </a>
+            </div>
             <div dir="ltr" class="fd-pagination-direction-override-display">
                 <ng-container *ngFor="let page of pages">
-                    <a
+                    <div
                         class="fd-pagination__link"
                         tabindex="0"
-                        role="button"
+                        role="option"
                         (keypress)="onKeypressHandler(page, $event)"
                         (click)="goToPage(page, $event)"
                         *ngIf="page !== -1; else more"
                         [attr.aria-selected]="currentPage === page"
+                        [attr.aria-label]="pageLabel + page"
```

Comment:
let's do `pageLabel + ' ' + page`, this currently works fine on chrome's screenreader but maybe not all of them

Reply:
Modified to use aria-current instead of aria-select as per the accessibility rules.